packages/libs/react-ui/src/components/NavHeader/NavHeaderNavigation.tsx

```diff
@@ -0,0 +1,56 @@
+import { NavGlow } from './assets/glow';
+import { glowClass, navListClass, navWrapperClass } from './NavHeader.css';
+import { INavHeaderLinkProps } from './NavHeaderLink';
+import useGlow from './useGlow';
+
+import React, { FC } from 'react';
+
+export type INavItemTarget = '_self' | '_blank';
+export interface INavItem {
+  href: string;
+  target?: INavItemTarget;
+  title: string;
+}
+export type INavItems = INavItem[];
+
+export interface INavHeaderNavigationProps {
+  children: React.ReactNode;
```

Comment:
Small improvement: You can type the children to be an array of a specific component 
```
export interface IBreadcrumbsProps {
  children?: FunctionComponentElement<IBreadcrumbItemProps>[];
  icon?: (typeof ProductIcon)[keyof typeof ProductIcon];
}
```